app/policies/article_policy.rb

```diff
@@ -178,6 +204,8 @@ def moderate?
 
   alias toggle_featured_status? revoke_publication?
 
+  alias can_adjust_any_tag? revoke_publication?
```

Comment:
@jeremyf So I decided to alias the check for an elevated user when it comes to tag-adjustment. The `revoke_publication` basically does the check, and I do not believe tags can be adjusted on an unpublished post.

Reply:
Love it!